utils/land-patterns.stanza

```diff
@@ -178,8 +180,8 @@ public defn soldermask-defined (p:Pad):
     pcb-pad SMD-p:
       name  = to-string("%_, SMD" % [name(p)])
       type  = pad-type(p)
-      shape = pad-shape(p)
-      apply-soldermask(0.85, true)
+      shape = offset(pad-shape(p), SOLDER-MASK-REGISTRATION)
```

Comment:
This fix is fine for now, but this utility is overly general and there are PRs coming that moves solder-mask-defined flag to within the individual landpattern generators.

Reply:
This should probably be replaced once we have a general purpose substitution mechanism as well.